actions/checks.loadbalancers.yaml

```diff
@@ -0,0 +1,15 @@
+description: Check for amphorae with an error status and find out if it is an issue with the amphora or loadbalancer
```

Comment:
So this should be a sensor, since we will likely trigger some sort of reactive action when Amphorae / the Load balancer falls over:
https://docs.stackstorm.com/sensors.html